iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergMetaHook.java

```diff
@@ -333,6 +334,12 @@ private static Properties getCatalogProperties(org.apache.hadoop.hive.metastore.
       properties.put(Catalogs.NAME, TableIdentifier.of(hmsTable.getDbName(), hmsTable.getTableName()).toString());
     }
 
+    hmsTable.getSd().getSerdeInfo().getParameters().entrySet().stream()
```

Comment:
`SerdeInfo::getParameters` has a `@Nullable` annotation. Shall we guard against that case to be safe?

Reply:
Good point. Added null check.